### PR TITLE
fix documentation for creating an HTTPScaledObject

### PR DESCRIPTION
**Pull request by iompo**
_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [x] Any necessary documentation is added, such as:
  - [`README.md`](/README.md)
  - [The `docs/` directory](./docs)
  - [The docs repo](https://github.com/kedacore/keda-docs)

Fixes #

### CODE REVIEW

File: docs/walkthrough.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You'll need to clone the repository to get access to this chart. If you have you
 
 ## Creating an `HTTPScaledObject`
 
-You interact with the operator via a CRD called `HTTPScaledObject`. This CRD object points the To get an example app up and running, read the notes below and then run the subsequent command from the root of this repository.
+You interact with the operator via a CRD called `HTTPScaledObject`. This CRD object points the request for an `Host` to a `Service`. To get an example app up and running, read the notes below and then run the subsequent command from the root of this repository.
```
Comment (arschles) [inline docs/walkthrough.md:23]: I added a suggestion below to clarify how the CRD works with the interceptor

```suggestion
You interact with the operator via a CRD called `HTTPScaledObject`. This CRD object instructs interceptors to forward requests for a given host to your app's backing `Service`. To get an example app up and running, read the notes below and then run the subsequent command from the root of this repository.
```

**Review by arschles — CHANGES_REQUESTED** [review]

**Comment by arschles** [comment]
@iompo are you still interested in getting this PR merged?

**Comment by iompo** [comment]
Hi @arschles sorry I totally forgot to apply the changes

**Review by arschles — APPROVED** [review]

**Comment by arschles** [comment]
Thanks for this @iompo!

**Pull request merged**